test/e2e/CallFunc.test.js

```diff
@@ -61,4 +60,32 @@ describe("callFunc", () => {
             "Warning calling function in CallFuncTestbed: no function interface specified for privateFunc",
         ]);
     });
+
+    test("components/call-func/scripts/throw/wrong-arg-type.brs", async () => {
+        await execute([resourceFile("throw", "wrong-arg-type.brs")], outputStreams);
+
+        expect(
+            allArgs(outputStreams.stderr.write)
+                .filter((arg) => arg !== "\n")
+                .map((arg) => arg.trim())
+        ).toEqual([
+            expect.stringContaining(
+                `test/e2e/resources/components/call-func/scripts/throw/wrong-arg-type.brs(5,4-46): Provided arguments don't match stronglyTyped's signature.
+    function stronglyTyped(arg1 as String) as String:
+        * Argument 'arg1' must be of type String, but received Object.`
```

Comment:
One of these days we'll pull in `ts-dedent` (https://www.npmjs.com/package/ts-dedent) and make these strings pretty 😜